newton/_src/viewer/viewer_gl.py

```
@@ -609,7 +608,7 @@ def get_frame(self, target_image: wp.array | None = None, render_ui: bool = Fals
 
         # Launch the RGB kernel.
         wp.launch(
-            warp.render.render_opengl.copy_rgb_frame_uint8,
+            warp._src.render.render_opengl.copy_rgb_frame_uint8,
```

Ruff (0.14.10)

Undefined name `warp`

(F821)